<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2020</width>
    <height>996</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database view</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="create_Frame">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>100</y>
      <width>551</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="create_btnRemote">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>REMOTE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="create_btnLocal">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>LOCAL</string>
     </property>
    </widget>
    <widget class="QLabel" name="create_labelOpen">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>391</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Open Database</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="query_Frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>881</width>
      <height>731</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="query_labelEditHint">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Query text</string>
     </property>
    </widget>
    <widget class="QPushButton" name="query_btnonExec">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Execute</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="query_Edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="query_btnOpen">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>70</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open another</string>
     </property>
    </widget>
    <widget class="QLabel" name="query_labelResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Query result</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="query_gridCols">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>861</width>
       <height>611</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2020</width>
     <height>39</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
